### PR TITLE
navigate to root on send or burn nft and confirmation drawer closes

### PR DESCRIPTION
**Pull request by tomlinton**
Closes https://github.com/coral-xyz/backpack/issues/1368 and some misc cleanup of unused functions/variables.

Blocked by figuring out https://github.com/coral-xyz/backpack/issues/1390

### CODE REVIEW

**Comment by tomlinton** [comment]
https://github.com/coral-xyz/backpack/issues/1390 is unrelated.

**Comment by armaniferrante** [comment]
App is now crashes after burning. Looking into it.

File: packages/app-extension/src/components/Unlocked/Nfts/Detail.tsx [hunk 1]
```diff
@@ -207,12 +193,19 @@ function SendScreen({ nft }: { nft: any }) {
...
+      // navigate back to the nav root because the send screen is no longer
+      // valid as the wallet no longer possesses the NFT.
+      if (!openConfirm && wasSent) {
+        await background.request({
+          method: UI_RPC_METHOD_NAVIGATION_TO_ROOT,
+          params: [],
+        });
+      }
+    })();
+  }, [openConfirm]);
```
Comment (armaniferrante) [inline packages/app-extension/src/components/Unlocked/Nfts/Detail.tsx:208]: Should we have added `wasSent` to the dependency array here?

Comment (tomlinton) [reply packages/app-extension/src/components/Unlocked/Nfts/Detail.tsx:208]: Yep my bad

**Comment by tomlinton** [comment]
Hrmm I did successfully test it, wonder if it was related to that other > 1 bug

**Pull request merged**